CRM/Upgrade/Incremental/php/FiveEleven.php

```diff
@@ -75,6 +75,18 @@ public function setPostUpgradeMessage(&$postUpgradeMessage, $rev) {
   public function upgrade_5_11_alpha1($rev) {
     $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
     $this->addTask('Update smart groups where jcalendar fields have been converted to datepicker', 'updateSmartGroups', $rev);
+    if (Civi::settings()->get('civimail_multiple_bulk_emails')) {
```

Comment:
@eileenmcnaughton shouldn't this be in 5_11_beta1 as it is in the RC?

Reply:
@seamuslee001 I guess - I can move it there if you think that is better

Reply:
@seamuslee001 ok updated